source/Hardpoint.cpp

```diff
@@ -223,6 +223,11 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	aim += angle;
 	start += aim.Rotate(outfit->HardpointOffset());
 
+	// Determine the inaccuracy to add to a new projectile
```

Comment:
This comment is just a statement of what the code does. Is it even necessary?

(Comments should explain _why_ not _how_, and if the "how" needs to be explained then there really better be a _why_, too.)

Reply:
I prefer clarity, personally.  Now, I can see where you're coming from and this isn't my personal project or anything so I'll gladly remove it.

Reply:
Removed